src/sentry/static/sentry/app/stores/hookStore.tsx

```diff
@@ -120,4 +120,6 @@ type HookStore = Reflux.Store & HookStoreInterface;
  */
 const HookStore = Reflux.createStore(hookStoreConfig) as HookStore;
 
+window.HookStore = HookStore;
```

Comment:
Let's do this in bootstrap.tsx

https://github.com/getsentry/sentry/blob/3f9c71202f641949a120528db4bc60a6ffc9838a/src/sentry/static/sentry/app/bootstrap.tsx#L184-L203